t/local_ingest.t

```diff
@@ -145,11 +150,61 @@ describe "HTFeed::PackageType::Simple::Download" => sub {
         packagetype => 'simple');
       my $download = $volume->get_sip_location();
       my $stage = HTFeed::PackageType::Simple::Download->new(volume => $volume);
-      $stage->{rclone} = "$FindBin::Bin/bin/rclone_stub.pl";
       $stage->run();
       ok($stage->succeeded() && -f $download);
     };
   };
 };
 
+describe "HTFeed::PackageType::Simple::Volume" => sub {
+  use HTFeed::PackageType::Simple::Download;
+  my $tmpdirs;
+  my $testlog;
+  my $save_rclone;
+  my $fetchdir;
+
+  before all => sub {
+    load_db_fixtures;
+    $tmpdirs = HTFeed::Test::TempDirs->new();
+    $testlog = HTFeed::Test::Logger->new();
+    set_config(0,'stop_on_error');
+    set_config($tmpdirs->test_home . "/fixtures/rclone_config.conf", 'rclone_config_path');
+  };
+
+  before each => sub {
+    $tmpdirs->setup_example;
+    $testlog->reset;
+    $save_rclone = $ENV{rclone};
+    $ENV{rclone} = "$FindBin::Bin/bin/rclone_stub.pl";
+    $fetchdir = $tmpdirs->dir_for("fetch");
+    set_config($fetchdir,'staging','fetch');
+    mkdir("$fetchdir/test");
+    system("touch","$fetchdir/test/test_objid.zip");
+    system("touch","$fetchdir/test/test_objid.xml");
+  };
+
+  after each => sub {
+    $tmpdirs->cleanup_example;
+    $ENV{rclone} = $save_rclone;
+    remove_tree($fetchdir);
+  };
+
+  after all => sub {
+    $tmpdirs->cleanup;
+  };
+
+  describe "#clean_sip_success" => sub {
+    it "calls rclone to remove SIP from Dropbox" => sub {
+      my $volume = HTFeed::Volume->new(
+        namespace => 'test',
+        objid => 'test_objid',
+        packagetype => 'simple');
+      eval {
+        $volume->clean_sip_success();
```

Comment:
The stub doesn't really do anything, so I think this test is just testing that `clean_sip_success` is defined.

It strikes me that now that `HTFeed::RClone` has its own class, we could mock that instead (e.g. using https://metacpan.org/pod/Test::Spec::Mocks) but I'm not sure it's worth the trouble given that we have the rclone stub  binary already. I suppose another option would be to have the rclone stub use its own temp directory for tests, and then it could actually implement delete.

Both options seem a little bit painful for what we'd gain given that we're really only testing one line of code (e.g. ensuring that rclone is called the way we expect) and that we aren't getting an integration test here, so I might have  talked myself into not worrying about testing this. I'm not sure where that leaves this test; I would be inclined just to remove it.

Reply:
I guess if we did inject a mock object into the stage / volume for use in the tests, we could use that for testing both the copy & the delete, which might be worthwhile; I think that would probably be preferable to continuing to build out `bin/rclone_stub.pl`. We could then perhaps test `RClone.pm` itself by injecting the command to run as `echo` or something.

Reply:
I like having the external binary because it throws a different codebase at sanity-checking the parameters, even though it's otherwise more or less of a no-op, and I think in that way it's more potent than an `echo`

Reply:
I would prefer to have the rclone binary set via config rather than environment variable, I think. (And it may make sense to do so given we have some configs like Jhove and Kakadu that have/need directions to find the executable. But may be overkill to require it.)

Reply:
Sounds reasonable. There's still the question of whether/how to test that `clean_sip_success` actually calls delete.

Reply:
I can at least check the log for a signal that the binary is called with a `delete` subcommand.